### PR TITLE
5146 - Fix on spinbox border issue in responsive view

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR fixes the spinbox and its border not properly rendered on responsive view.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5146

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/form/example-inputs.html
- Open Developer Console
- Make it responsive to 1577 x 836 50% or 75%

**Included in this Pull Request**:
~~- [ ] An e2e or functional test for the bug or feature.~~
- [x] A note to the change log.

### CODE REVIEW

**Review by ericangeles — CHANGES_REQUESTED** [review]
@tjamesallen15 , looks like there's a failing visual regress. We can talk about the fix offline.

File: src/components/input/_input-mixins.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@
   text-align: left;
   width: 300px;
 
-  &:hover {
+  &:not(.spinbox):hover {
```
Comment (ericangeles) [inline src/components/input/_input-mixins.scss:23]: This probably causes visual regress. You might want to override it in the spinbox CSS file specifically.

**Review by ericangeles — CHANGES_REQUESTED** [review]

**Comment by tmcconechy** [comment]
Screen Shot 2021-06-01 at 9 23 18 AM

Also seeing at this dimension that it has a strange lighter border on the buttons

**Comment by tjamesallen15** [comment]
Hello @tmcconechy , 
I will check this one. Thanks!

**Pull request merged**